src/pages/CompanyCourseManagement/components/TrainingListItem.jsx

```diff
@@ -0,0 +1,53 @@
+import { useTheme } from '@emotion/react';
+import { Grid, ListItem, ListItemText } from '@mui/material';
+import PropTypes from 'prop-types';
+
+function TrainingListItem({ trainingName, trainingCategory }) {
+  const theme = useTheme();
+
+  return (
+    <Grid item xs={6}>
+      <ListItem
+        style={{
+          height: '150px',
```

Comment:
```suggestion
          height: '100px',
```

use a height of 100px to have it identical to jobProfile